src/controllers/tasks/active.officers.controller.ts

```diff
@@ -8,19 +8,26 @@ import { ActiveOfficerDetails } from "private-api-sdk-node/dist/services/confirm
 import { getActiveOfficerDetailsData } from "../../services/active.officer.details.service";
 import { formatTitleCase } from "../../utils/format";
 
+let activeOfficerDetails: ActiveOfficerDetails;
```

Comment:
don't think we can use global variables, get and post will be coming in on different requests, also with scaling/load balancing, get and post could happen on different servers